test/distributed_exec.jl

```diff
@@ -1569,6 +1569,41 @@ catch ex
     @test ex.captured.ex.exceptions[2].ex == UndefVarError(:DontExistOn1)
 end
 
+@test let
+    # creates a new worker in the same folder and tries to include file on both procs
+    try
+        touch("temp.jl")
```

Comment:
we should avoid assuming the current working directory is writable, in some installations it won't be - a `mktemp() do` block might work well here